programs/stake/src/stake_state.rs

```diff
@@ -1000,10 +1002,17 @@ struct ValidatedDelegatedInfo {
 fn validate_delegated_amount(
     account: &BorrowedAccount,
     meta: &Meta,
+    feature_set: &FeatureSet,
 ) -> Result<ValidatedDelegatedInfo, InstructionError> {
     let stake_amount = account
         .get_lamports()
         .saturating_sub(meta.rent_exempt_reserve); // can't stake the rent
+    if feature_set.is_active(&feature_set::raise_minimum_stake_delegation_to_1_sol::id())
+        && stake_amount < crate::get_minimum_delegation(feature_set)
+    {
+        return Err(InstructionError::InsufficientStakeDelegation);
+    }
+
```

Comment:
🚨 New behavior! 🚨

(Tyera, I think you commented over in Jon's PR to not make this an InstructionError, yes? It's likely that Jon's PR will merge first, so that'll fix this bit here.)